recipes/pybind11-numpy-example/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pybind11-numpy-example" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pybind11_numpy_example-{{ version }}.tar.gz
+  sha256: 9bab0607c9697673fd1bd6dc3b09689a10cbd8c908aa062298b4ba3801c224b4
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+    - ninja
+  host:
+    - numpy
+    - pip
+    - pybind11
+    - python
+    - scikit-build-core
+  run:
+    - python
```

Comment:
The build section is for build tools and the stdlib template.

The host section is for things that must match the target platform when cross-compiling.

I have moved numpy to the host section because you probably use the numpy headers during compiling and need the ABI to match at build and runtime.

Reply:
thanks!